### PR TITLE
Change upload path to /compliance/

### PR DESCRIPTION
**Pull request by isimluk**
Fixes OpenSCAP/foreman_openscap#44

Waits on OpenSCAP/foreman_openscap#44

### CODE REVIEW

File: lib/smart_proxy_openscap/openscap_lib.rb [hunk 1]
```diff
@@ -109,7 +109,7 @@ def forward_date_dir(cname, policy_name, date, date_dir)
     end
 
     def upload_path(cname, policy_name, date)
-      return "/api/v2/openscap/arf_reports/#{cname}/#{policy_name}/#{date}"
+      return "/api/v2/compliance/arf_reports/#{cname}/#{policy_name}/#{date}"
```
Comment (shlomizadok) [inline lib/smart_proxy_openscap/openscap_lib.rb]: Should be policy_id ?

Comment (isimluk) [inline lib/smart_proxy_openscap/openscap_lib.rb]: Right.

**Comment by isimluk** [comment]
Renamed. Please re-review.

File: lib/smart_proxy_openscap/openscap_lib.rb [hunk 2]
```diff
@@ -32,10 +32,10 @@ def self.common_name(request)
     return cn
   end
 
-  def self.spool_arf_dir(common_name, policy_name)
-    validate_policy_name policy_name
+  def self.spool_arf_dir(common_name, policy_id)
+    validate_policy_id policy_id
```
Comment (shlomizadok) [inline lib/smart_proxy_openscap/openscap_lib.rb]: parentheses (https://github.com/styleguide/ruby)

File: lib/smart_proxy_openscap/openscap_lib.rb [hunk 3]
```diff
@@ -59,9 +59,9 @@ def self.send_spool_to_foreman
   end
 
   private
-  def self.validate_policy_name name
-    unless /[\w-]+/ =~ name
-      raise Proxy::Error::BadRequest, "Malformed policy name"
+  def self.validate_policy_id id
```
Comment (shlomizadok) [inline lib/smart_proxy_openscap/openscap_lib.rb]: ditto (parentheses)

**Comment by shlomizadok** [comment]
lgtm, besides the parentheses.

**Comment by isimluk** [comment]
Merged as 4ade0699c4c02039550ea7cacb9d94630e5d11dd and c39d358d9d8c7487482df409f18dfda7fb070a15.

Will fix parenthesis later on ;-)

Thanks!

**Pull request merged**